CRM/Contact/Form/Task/Email.php

```diff
@@ -131,11 +131,7 @@ public function preProcess() {
     if (!$cid && $this->_context != 'standalone') {
       parent::preProcess();
     }
-
-    //early prevent, CRM-6209
-    if (count($this->_contactIds) > CRM_Contact_Form_Task_EmailCommon::MAX_EMAILS_KILL_SWITCH) {
-      CRM_Core_Error::statusBounce(ts('Please do not use this task to send a lot of emails (greater than %1). We recommend using CiviMail instead.', array(1 => CRM_Contact_Form_Task_EmailCommon::MAX_EMAILS_KILL_SWITCH)));
-    }
+    CRM_Contact_Form_Task_EmailCommon::bounceIfSimpleMailLimitExceeded(count($this->_contactIds));
```

Comment:
Hmm perhaps but it's called 'Send Email' here https://docs.civicrm.org/user/en/stable/email/what-you-need-to-know/ - it feels like scope creep to rebrand that feature & I'm a bit reluctant to entrench a user-term in the code if it's not accepted at the user level yet.

I could change all those code places to 'send_email' to match the docs - but not sure that is clearer than simple_mail